### ISSUE TITLE
API key location

### ISSUE THREAD

**Issue by kenthinson** [UnderMybrella/EternalJukebox#42]
Hello I have cloned the repo and have sucessfully compiled the program. I get API errors. So I went back and was following the readme. I have gotten the Spotify API and youtube API keys. However I dont know where to put them. Can you please update the readme with more information on this? thank you.

**Comment by stiggy87** [comment]
These need to go into the config.json (other forked versions explain this one).

**Comment by kenthinson** [comment]
It would be good to add it to the readme? Or show inside the config file with an example?

**Comment by stiggy87** [comment]
You're not wrong, but I am not the one maintaining this repo, but there's another issue about this same thing.

**Comment by kenthinson** [comment]
Ah thanks. I'll look into it when I am not in transit. Using the phone with GitHub is not so easy. :)
